jerry-core/parser/js/js-lexer.c

```diff
@@ -309,37 +309,36 @@ typedef struct
  * Keyword defines
  */
 #define LEXER_KEYWORD(name, type) { (const uint8_t *) (name), (type) }
-#define LEXER_KEYWORD_END()       { (const uint8_t *) NULL, LEXER_EOS }
+#define LEXER_KEYWORD_LIST_LENGTH(name) (const uint8_t) (sizeof ((name)) / sizeof ((name)[0]))
+#define LEXER_KEYWORD_END()
```

Comment:
The `LEXER_KEYWORD_END()`  is still needed?

Reply:
Good catch! It's a leftover code.